eZ/Publish/Core/Repository/TrashService.php

```diff
@@ -348,6 +348,7 @@ protected function buildDomainTrashItemObject(Trashed $spiTrashItem, Content $co
                 'depth' => $spiTrashItem->depth,
                 'sortField' => $spiTrashItem->sortField,
                 'sortOrder' => $spiTrashItem->sortOrder,
+                'trashed' => isset($spiTrashItem->trashed) ? new DateTime('@' . $spiTrashItem->trashed) : new DateTime(),
```

Comment:
`new DateTime(),` will mean `now`, to stay in sync with database lets fall back to unixtime 0.

Reply:
@thiagocamposviana Can you make this change? Looks like this is the only thing stopping this feature from being approved.